picamera2/picamera2.py

```diff
@@ -19,6 +19,7 @@
 from enum import Enum
 import piexif
 
+from picamera2.controls import Controls
```

Comment:
I know I like to separate my commits nicely, but I think this one could be combined with the first commit that uses the new controls!!

Reply:
I started to think that after the first few comments. Just did that!